src/Data/Integer/Divisibility/Signed.agda

```diff
@@ -50,38 +49,38 @@ open _∣_ using (quotient) public
 ∣ᵤ⇒∣ {k} {i} (divides 0           eq) = divides (+ 0) (∣i∣≡0⇒i≡0 eq)
 ∣ᵤ⇒∣ {k} {i} (divides q@(ℕ.suc _) eq) with k ≟ +0
 ... | yes refl = divides +0 (∣i∣≡0⇒i≡0 (trans eq (ℕ.*-zeroʳ q)))
-... | no  neq  = divides (sign i S.* sign k ◃ q) (◃-cong sign-eq abs-eq)
+... | no  neq  = divides (sign i Sign.* sign k ◃ q) (◃-cong sign-eq abs-eq)
   where
-  ikq = sign i S.* sign k ◃ q
+  ikq = sign i Sign.* sign k ◃ q
 
   *-nonZero : ∀ m n .{{_ : ℕ.NonZero m}} .{{_ : ℕ.NonZero n}} → ℕ.NonZero (m ℕ.* n)
   *-nonZero (ℕ.suc _) (ℕ.suc _) = _
 
   ◃-nonZero : ∀ s n .{{_ : ℕ.NonZero n}} → NonZero (s ◃ n)
-  ◃-nonZero S.- (ℕ.suc _) = _
-  ◃-nonZero S.+ (ℕ.suc _) = _
+  ◃-nonZero Sign.- (ℕ.suc _) = _
+  ◃-nonZero Sign.+ (ℕ.suc _) = _
 
   ikq≢0 : NonZero ikq
-  ikq≢0 = ◃-nonZero (sign i S.* sign k) q
+  ikq≢0 = ◃-nonZero (sign i Sign.* sign k) q
 
   instance
     ikq*∣k∣≢0 : ℕ.NonZero (∣ ikq ∣ ℕ.* ∣ k ∣)
     ikq*∣k∣≢0 = *-nonZero ∣ ikq ∣ ∣ k ∣ {{ikq≢0}} {{≢-nonZero neq}}
 
   sign-eq : sign i ≡ sign (ikq * k)
   sign-eq = sym $ begin
-    sign (ikq * k)                  ≡⟨ sign-◃ (sign ikq S.* sign k) (∣ ikq ∣ ℕ.* ∣ k ∣) ⟩
-    sign ikq S.* sign k             ≡⟨ cong (S._* sign k) (sign-◃ (sign i S.* sign k) q) ⟩
-    (sign i S.* sign k) S.* sign k  ≡⟨ SProp.*-assoc (sign i) (sign k) (sign k) ⟩
-    sign i S.* (sign k S.* sign k)  ≡⟨ cong (sign i S.*_) (SProp.s*s≡+ (sign k)) ⟩
-    sign i S.* S.+                  ≡⟨ SProp.*-identityʳ (sign i) ⟩
+    sign (ikq * k)                  ≡⟨ sign-◃ (sign ikq Sign.* sign k) (∣ ikq ∣ ℕ.* ∣ k ∣) ⟩
```

Comment:
picky: the alignment seems to need tweaked.

Reply:
Revised first commit fixes the alignment, but I wonder if this is a case where interleaving the terms and the reasoning line-by-line might be a better layout for the sake of avoiding long lines...?

Reply:
The alignment in `Data.List.Relation.Binary.BagAndSetEquality` is also (historically) terrible, and I don't necessarily think this worth fixing here; against #1630 / #1930 , to my surprise this module also makes extensive use of `inspect`... so might want a separate consideration outside this 'boilerplate' PR...

Reply:
Indeed: a global `cosmetic` issue on 'fixing alignment in reasoning' might be in order...